src/components/ReservationList.jsx

```diff
@@ -0,0 +1,111 @@
+import React, { useState, useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import '../styles/MyReservations.css';
+import fetchDelete from '../features/deleteSlice';
+import SignOutButton from './SignOutButton';
+
+function ReservationList() {
```

Comment:
- [ ]  Please, kindly refactor this functional componenent to ES6 arrow function, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const ReservationList= () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

That way, your code will meet the latest standards. 💪

Reply:
@Meltrust Thank you for your helpful review we will make changes as per your request